code/game/atoms_movable.dm

```diff
@@ -719,3 +719,46 @@
 
 /atom/movable/proc/can_apply_inertia()
 	return (!src.anchored && !(src.pulledby && src.pulledby.Adjacent(src)))
+
+/atom/movable/proc/send_to_future(var/duration)
+	var/init_invisibility = invisibility
+	var/init_invuln = flags & INVULNERABLE
+	var/init_density = density
+	var/init_anchored = anchored
+	var/init_timeless = flags & TIMELESS
+	var/init_blinded
+	var/mob/M
+	if(ismob(src))
```

Comment:
>but anything overriding it can wrap code around the ..() without issues

If a parent with `set waitfor = 0` is called via `..()`, do the sleeps within it halt the calling child?
Because if not then I don't see how I can have the override execute behavior after the `..()` at the correct time, since it won't wait for the parent proc to finish before continuing.